bin/lxa-iobus-server

```diff
@@ -103,7 +103,8 @@ log_level = {
 logging.basicConfig(level=log_level)
 
 # setup server
-loop = asyncio.get_event_loop()
+loop = asyncio.new_event_loop()
+asyncio.set_event_loop(loop)
```

Comment:
I am not an experience python version historian, so i can't get `pyenv` to work right now, but maybe you would like to test my latest change.
I've added a call to `asyncio.set_event_loop()` in the hope that following `get_event_loop()`-calls, like for example in [`asyncio.Queue()`](https://github.com/python/cpython/blob/3.7/Lib/asyncio/queues.py#L34) will then no now longer start a new event loop. Which otherwise results in general confusion.

As I said. This is completely untested.

Reply:
Tested on 3.7, 3.9 and 3.10. Works fine.
Thanks!